telegram/ext/jobqueue.py

```diff
@@ -208,22 +242,36 @@ class Job(object):
         callback (function): The callback function that should be executed by the Job. It should
             take two parameters ``bot`` and ``job``, where ``job`` is the ``Job`` instance. It
             can be used to terminate the job or modify its interval.
-        interval (float): The interval in which this job should execute its callback function in
-            seconds.
+        interval (float or datetime.time): The interval in which this job should execute its
```

Comment:
I believe you switched to `timedelta` instead of `time`, please update this docstring to reflect that change.